articles/user-guide/libraries/additional-libraries.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Using additional Q# libraries
+description: Learn how to add the Microsoft Quantum numerics library to your Visual Studio 2019 or later installation. 
```

Comment:
That's right; numerics is the easiest to demonstrate through pure Q#, and so it makes a nice example for how to add libraries to a project. As a side note, I've avoided using the word "install" here since it seems to cause some confusion for users. Libraries don't require installation, as they are used on a project-local basis (either added to csproj files or using `%package` commands). The chemistry library works well with additionally installed tools, but then that's not installing the library itself.